src/components/BookList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import BookIndividual from './BookIndividual';
+
+const BookList = ({ books, onDelete }) => (
+  <div>
+    {books.map((book) => (
+      <BookIndividual key={book.id} book={book} onDelete={onDelete} />
+    ))}
+  </div>
+);
```

Comment:
- [ ] Great job working on the required changes but I think it would be great if you render a list of static books I could see that you made the functionality for the button but to stick with the requirements you should render a static list as mentioned by the previous reviewer so kindly add that 💯✨🚀

![image](https://github.com/gilberthappi/Bookstore-App/assets/58553711/86038ba1-fac7-4288-960e-c78ca88a8375)